### PR TITLE
language-clients: Improve xpath query documentation

### PR DESCRIPTION
**Pull request by abeaumont**
Signed-off-by: Alfredo Beaumont <alfredo.beaumont@gmail.com>

### CODE REVIEW

File: user/language-clients.md [hunk 1]
```diff
@@ -1,26 +1,27 @@
 # Babelfish clients
 
-Over the network protocol and (soon) using
-[`libuast`](https://github.com/bblfsh/libuast) there are some clients that allow
-you to use Babelfish with a higher level API, abstracting some of the complexities
-of [gRPC](https://grpc.io) communication, parsing, and the [base C `libuast`
-API](https://github.com/bblfsh/libuast/tree/master/src).
+There are some clients in different languages that provides a higher level API,
```
Comment (juanjux) [inline user/language-clients.md]: that provide

File: user/language-clients.md [hunk 2]
```diff
@@ -45,9 +46,64 @@ func main() {
...
+	}
 }
 ```
 
-Currently we're integrating [`libuast`](https://github.com/bblfsh/libuast) into
-the clients which will greatly increase the capabilities of the clients adding
-[XPath](https://en.wikipedia.org/wiki/XPath)-like querying of the UAST tree.
+## Query language
+
+[`libuast`](https://github.com/bblfsh/libuast) provides a query language for UASTs.
```
Comment (juanjux) [inline user/language-clients.md]: "That provides a `XPath-like` query language" maybe?

It mentions XML below but without that context people could wonder why we're mapping things to a XML node.

Comment (abeaumont) [reply user/language-clients.md]: right, should provide a reference to xpath too

**Comment by abeaumont** [comment]
@juanjux fixed

**Pull request merged**